### ISSUE TITLE
windows support with firebase_core_desktop

### ISSUE THREAD

**Issue by somebay** [flyerhq/flutter_firebase_chat_core#99]
the Flutter team has announced that Flutter on desktop platforms moved to the stable channel, which includes Linux and MacOS. Windows was announced a couple of months back.

i have folllowed this intructions but without success,
https://invertase.io/blog/announcing-flutterfire-desktop-auth-stable

is there any plans to support firebase_core_desktop for windows applications?

**Comment by demchenkoalex** [comment]
No, I don't think so.. this package is more like an example on how the UI counterpart works, and you can easily do desktop yourself if needed I guess.
